spec/models/availability_spec.rb

```diff
@@ -34,4 +34,17 @@
       expect(availability.partially_available?(event)).to eq(false)
     end
   end
+
+  context 'cancel_duplicates' do
+    it 'sets status for previous availabilities with matching start and end times to Cancelled' do
```

Comment:
Would you mind rearranging the phrasing of this to say "sets status to X for..."? It took me a few reads before realizing what it was saying. 😬